.generate/gen-all.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+#  Copyright 2019 The Operator-SDK Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 test.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+
+function header_text {
+  echo "$header$*$reset"
+}
+
+HELM_SCRIPT_PATH="../helm/.generate/gen-helm-memcached.sh"
+GO_SCRIPT_PATH="../go/.generate/gen-go-sample.sh"
+ANS_SCRIPT_PATH="../ansible/.generate/gen-ansible-memcached.sh"
+
+RELEASE_VERSION=v1.0.0
+
+header_text "downloading binaries"
+
+curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
```

Comment:
Refactor these into a function:
```sh
function install_bin() {
	local url="https://github.com/operator-framework/operator-sdk/releases/download/${2}/${1}-${2}-x86_64-linux-gnu"
	curl -sSLo $1 $url
	chmod +x $1
	...
}

install_bin operator-sdk $RELEASE_VERSION
install_bin helm-operator $RELEASE_VERSION
install_bin ansible-operator $RELEASE_VERSION

...
```

Reply:
IMO cannot be based for Linux. It needs work in Mac OS as well.

Reply:
@estroz and @theishshah, 

The way that it is now it cannot work in a Mac. See my above suggestion to fix it.